iaso/tests/api/test_profiles_bulk_update.py

```diff
@@ -684,80 +723,154 @@ def test_profile_bulkupdate_select_all(self):
         self.assertJSONResponse(response, 201)
         data = response.json()
         task = self.assertValidTaskAndInDB(data["task"], status="QUEUED", name="profiles_bulk_update")
-        self.assertEqual(task.launcher, self.yoda)
+        self.assertEqual(task.launcher, self.user_admin)
 
         # Run the task
         self.runAndValidateTask(task, "SUCCESS")
-        self.raccoon.refresh_from_db()
-        self.assertEqual(self.raccoon.iaso_profile.language, "en")
-        self.root.refresh_from_db()
-        self.assertEqual(self.root.iaso_profile.language, "en")
-        self.wolverine.refresh_from_db()
-        self.assertEqual(self.wolverine.iaso_profile.language, "en")
-        self.chewie.refresh_from_db()
-        self.assertEqual(self.chewie.iaso_profile.language, "fr")
-        self.luke.refresh_from_db()
-        self.assertEqual(self.luke.iaso_profile.language, "fr")
-        self.yoda.refresh_from_db()
-        self.assertEqual(self.yoda.iaso_profile.language, "fr")
-        self.assertEqual(7, am.Modification.objects.count())
-        self.obi_wan.refresh_from_db()
-        self.assertEqual(self.obi_wan.iaso_profile.language, "fr")
-        self.assertEqual(7, am.Modification.objects.count())
+        self.user_admin_account2.refresh_from_db()
+        self.assertEqual(self.user_admin_account2.iaso_profile.language, "en")
+        self.user_admin_no_task_account2.refresh_from_db()
+        self.assertEqual(self.user_admin_no_task_account2.iaso_profile.language, "en")
+        self.user_admin_account3.refresh_from_db()
+        self.assertEqual(self.user_admin_account3.iaso_profile.language, "en")
+        self.user_admin_no_task2.refresh_from_db()
+        self.assertEqual(self.user_admin_no_task2.iaso_profile.language, "fr")
+        self.user_admin_no_task.refresh_from_db()
+        self.assertEqual(self.user_admin_no_task.iaso_profile.language, "fr")
+        self.user_admin.refresh_from_db()
+        self.assertEqual(self.user_admin.iaso_profile.language, "fr")
+        self.assertEqual(8, am.Modification.objects.count())
+        self.user_managed.refresh_from_db()
+        self.assertEqual(self.user_managed.iaso_profile.language, "fr")
+        self.assertEqual(8, am.Modification.objects.count())
```

Comment:
Is this change due to the adding of teams to the test setup?

Reply:
You mean the count of users? I added a user that wasn't there in the initial setup: `user_admin_no_task_account2` if I remember correctly